docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -190,6 +190,12 @@ To upgrade an existing Ansible installation in this Python environment to the la
 
     $ python3 -m pip install --upgrade --user ansible
 
+Installing Ansible to containers
+================================
+
+Instead of installing Ansible content manually, you can simply build an execution environment container image or use one of available community images as your control node.
```

Comment:
```suggestion
Instead of installing Ansible content manually, you can simply build an execution environment container image or use one of the available community images as your control node.
```